docs/sources/flow/reference/components/otelcol.processor.transform.md

```diff
@@ -73,9 +73,9 @@ to a new metric data type or can be used to create new metrics.
 - [Orphaned Telemetry][]: The processor allows you to modify `span_id`, `trace_id`, and `parent_span_id` for traces 
   and `span_id`, and `trace_id` logs.  Modifying these fields could lead to orphaned spans or logs.
 
-[Unsound Transformations]: https://github.com/open-telemetry/opentelemetry-collector/blob/v0.85.0/docs/standard-warnings.md#unsound-transformations
-[Identity Conflict]: https://github.com/open-telemetry/opentelemetry-collector/blob/v0.85.0/docs/standard-warnings.md#identity-conflict
-[Orphaned Telemetry]: https://github.com/open-telemetry/opentelemetry-collector/blob/v0.85.0/docs/standard-warnings.md#orphaned-telemetry
+[Unsound Transformations]: https://github.com/open-telemetry/opentelemetry-collector/blob/v0.87.0/docs/standard-warnings.md#unsound-transformations
```

Comment:
Could be in a separate PR, but you could change the header block in `docs/sources/_index.md.t` (and then run `make generate-versioned-files`) to have:
```yaml
cascade:
  AGENT_RELEASE: v0.37.2
  OTEL_VERSION: v0.86.0
```
And then change these links to:
```suggestion
[Unsound Transformations]: https://github.com/open-telemetry/opentelemetry-collector/blob/{{< param "OTEL_VERSION" >}}/docs/standard-warnings.md#unsound-transformations
```
And now you only have to update it in one place and it will propagate :smiley:

I'd also be happy to do this in a followup PR, if you'd prefer

Reply:
Thank you, that's a great idea! I pushed such a change now.